packages/reports/addon/components/power-select-bulk-import-trigger.js

```diff
@@ -68,9 +68,9 @@ export default Trigger.extend({
 
       if (isBulkImportRequest) {
         setProperties(this, {
-          _showBulkImport: true,
-          _bulkImportQueryIds: queryIds,
-          _bulkImportRawValue: pastedData
+          //_showBulkImport: true,
+          //_bulkImportQueryIds: queryIds,
+          //_bulkImportRawValue: pastedData
```

Comment:
Can we either change the function name to `_onPaste` or comment out
https://github.com/yahoo/navi/blob/b4736f74e3eca8d0336ee2521fc71b5bb57773b3/packages/reports/addon/templates/components/power-select-bulk-import-trigger.hbs#L39

Then add a TODO somewhere in JSDoc saying paste has been disabled while bulk import is not working